### DISCUSSION TITLE
My request had been approved 3 days ago. Still not able to access the benefits

### DISCUSSION THREAD

**Discussion by UdhavSharma2004** [community/community#142297] [GitHub Education] [answered]
### Hi everyone,

My request had been approved 3 days ago. Still not able to access the benefits.
Check the credentials below.
Please help me to know what possible could be done.
![image](https://github.com/user-attachments/assets/cc089cb1-456e-4526-83e0-2020fd044cb7)

**Comment by davevad93** [comment]
Hi @UdhavSharma2004 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @UdhavSharma2004 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by ojmarcelino** [comment]
First things first, congrats on being approved.
An easy and simple way to check is browsing through [Settings \ Profile](https://github.com/settings/profile), and check if "Display PRO Badge" is available.

Source: [Displaying badges on your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile)

**Comment by tharun977** [comment]
It seems like your benefits haven't been activated yet, even after your request was approved 3 days ago. Delays like this can happen, and benefits may take a bit more time to appear. You could try checking the Founders Hub portal or your GitHub account settings to see if there's any update. If nothing shows up, contacting GitHub or Microsoft for Startups support directly would be the best next step to resolve the issue.

**Comment by UdhavSharma2004** [comment, marked as answer]
Thanks for the suggestions, I have got the Github Pro Pack as a student.

Thanks once again.
